sdk/nodejs/cmd/pulumi-language-nodejs/main.go

```diff
@@ -1125,7 +1127,7 @@ func (host *nodeLanguageHost) RuntimeOptionsPrompts(ctx context.Context,
 			Key:         "packagemanager",
 			Description: "The package manager to use for installing dependencies",
 			PromptType:  pulumirpc.RuntimeOptionPrompt_STRING,
-			Choices:     plugin.MakeExecutablePromptChoices("npm", "pnpm", "yarn"),
+			Choices:     plugin.MakeExecutablePromptChoices("bun", "npm", "pnpm", "yarn"),
```

Comment:
Probably should leave npm as first choice?

Reply:
I agree - wasn't sure if it should be alphabetical order or not